src/backend/TrafficCourts/Citizen.Service/Models/Dispute/NoticeOfDispute.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Swashbuckle.AspNetCore.Annotations;
+using System.Text.Json.Serialization;
+using TrafficCourts.Common.Converters;
+
+namespace TrafficCourts.Citizen.Service.Models.Dispute
+{
+    public class NoticeOfDispute
+    {
+        public string? TicketNumber { get; set; }
+        public string? ProvincialCourtHearingLocation { get; set; }
+        public DateTime? IssuedDate { get; set; }
+        public string? Surname { get; set; }
+        public string? GivenNames { get; set; }
+        public string? Address { get; set; }
+        public string? City { get; set; }
+        public string? Province { get; set; }
+        public string? PostalCode { get; set; }
+        public string? HomePhoneNumber { get; set; }
+        public string? WorkPhoneNumber { get; set; }
+        public string? EmailAddress { get; set; }
+        public IList<DisputedCount> DisputedCounts { get; set; } = new List<DisputedCount>();
+        public bool RepresentedByLawyer { get; set; }
+        public LegalRepresentation? legalRepresentation { get; set; }
+        public string? InterpreterLanguage { get; set; }
+        public int NumberOfWitness { get; set; }
+        public string? FineReductionReason { get; set; }
+        public string? TimeToPayReason { get; set; }
+        public bool CitizenDetectedOcrIssues { get; set; }
+        public ViolationTicket ViolationTicket { get; set; } = new();
```

Comment:
I am still not convinced we should be accepting "ticket" information from the citizen, Either they lookup the ticket or they OCR their ticket. In both cases we cache the data and use that as "the source of truth"

Reply:
Changed the approach to persist Violation Ticket details and had the looked up tickets to be saved in Redis Cache so the Dispute create request only requires Ticket ID key to be provided to save a dispute along with ticket details.